Formula/blucat.rb

```diff
@@ -17,6 +17,13 @@ def install
   end
 
   test do
-    system "#{bin}/blucat", "doctor"
+    begin
+      io = IO.popen(bin/"blucat", "scan", "0")
```

Comment:
I expected this to look like `[bin/"blucat", "scan", "0"]` but maybe i'm confused